lib/node_modules/@stdlib/blas/base/snrm2/test/test.ndarray.native.js

```diff
@@ -110,6 +136,23 @@ tape( 'if provided an `N` parameter less than or equal to `0`, the function retu
 	t.end();
 });
 
+tape( 'the function returns NaN if provided an array element equal to NaN', function test( t ) {
```

Comment:
```suggestion
tape( 'the function returns NaN if provided an array element equal to NaN', opts, function test( t ) {
```

You need to include the `opts` argument in `*.native.js` tests in order to guard against the scenario where the native add-on has not been built.